Gemfile

```
@@ -19,7 +19,7 @@ group :development, :test do
   # auto-load factories from spec/factories
   gem 'factory_bot_rails'
 
-  gem 'rails', '~>  7.0'
+  gem 'rails', '~>  7.1.0'
```

We still want to test 7.0.x here - since that's what will actually be shipped to customers

```suggestion
  gem 'rails', '~> 7.0.8'
```

We might want to lock in a better pattern moving forward though, since this kind of edge-case will happen again in the future. i.e. we should really be testing against multiple versions of Rails, so thata we can detect issues sooner?